modules/iam-role-for-service-accounts-eks/variables.tf

```
@@ -145,6 +145,19 @@ variable "attach_efs_csi_policy" {
   default     = false
 }
 
+# S3 CSI
+variable "attach_s3_csi_policy" {
+  description = "Determines whether to attach the S3 CSI IAM policy to the role"
+  type        = bool
+  default     = false
+}
+
+variable "s3_csi_bucket_arns" {
+  description = "S3 bucket ARNs to allow S3 CSI to manage buckets"
+  type        = list(string)
+  default     = ["arn:aws:s3:::*"]
```

with the change above, this becomes
```suggestion
  default     = []
```

don't forget to add the new `mountpoint_s3_csi_path_arns` variable as well and ensure the descriptions detail the differences